File: project2/Team1/spring/src/main/java/bitc/full502/spring/dto/LodgingBookingDto.java
package bitc.full502.spring.dto;

import lombok.Data;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Data
public class LodgingBookingDto {
    private Long id;          // ← 추가: 예약 PK
    private Long userId;
    private Long lodId;
    private String ckIn;
    private String ckOut;
    private Integer adult;
    private Integer child;
    private String roomType;
    private Long totalPrice;
    private String status;

    // ↓↓↓ 목록/상세 카드에 쓸 숙소 정보 추가
    private String lodName;
    private String lodImg;
    private String addrRd;
    private String addrJb;
}

File: project2/Team1/android/app/src/main/java/bitc/fullstack502/android_studio/network/dto/LodgingBookingDto.kt
package bitc.fullstack502.android_studio.network.dto

data class LodgingBookingDto(
    val id: Long? = null,       // 예약 PK
    val userId: Long,
    val lodId: Long,
    val ckIn: String,
    val ckOut: String,
    val totalPrice: Long,
    val roomType: String,
    val adult: Int,
    val child: Int,
    val status: String,

    // 서버에서 내려주는 숙소 정보
    val lodName: String?,       // 숙소 이름
    val lodImg: String?,        // 숙소 이미지
    val addrRd: String?,        // 도로명 주소
    val addrJb: String?         // 지번 주소
)
